packages/vetrina/src/Vetrina/Vetrina.purs

```diff
@@ -98,6 +96,7 @@ data OrderFailure
   | AuthenticationError
   | ServerError
   | UnrecognizedError String
+  | UnexpectedError
```

Comment:
Let's either have `UnrecognizedError String` or `UnexpectedError`, but not both.

Reply:
Semantically they are slightly different, what would be the criterion to decide? Maybe what's the more general? In that case I would go with `UnexpectedError`.

Reply:
Yeah, maybe with the message as well: `UnexpectedError String`. What do you think?

Reply:
I agree 👍